src/main/java/org/folio/service/BaseService.java

```diff
@@ -200,19 +200,19 @@ public CompletableFuture<Void> handlePutRequest(String endpoint, JsonObject reco
   public CompletableFuture<Void> handleDeleteRequest(String endpoint, HttpClientInterface httpClient,
       Map<String, String> okapiHeaders, Logger logger) {
     CompletableFuture<Void> future = new CompletableFuture<>();
-    logger.debug(CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpoint);
+    logger.debug("Trying to delete '{}' object", endpoint);
     try {
       httpClient.request(HttpMethod.DELETE, endpoint, okapiHeaders)
         .thenAccept(this::verifyResponse)
         .thenApply(future::complete)
         .exceptionally(t -> {
-          String errorMessage = String.format(EXCEPTION_CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpoint);
-          logger.error(errorMessage, t);
+          // logger.error("Error deleting '{}' object", endpoint, e);
+          logger.error("Delete '{}' request failed", endpoint, t);
           future.completeExceptionally(t);
           return null;
         });
     } catch (Exception e) {
-      logger.error(String.format(EXCEPTION_CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpoint), e);
+      logger.debug("Error deleting object by endpoint '{}'", endpoint, e);
       future.completeExceptionally(e);
```

Comment:
1 . Should I use two loggers in both parts
` exceptionally(t -> `
`catch(e) {`  or just one part
2. Which is a more useful loge 
`logger.error("Error deleting '{}' object", endpoint, e);`
`logger.error("Delete '{}' request failed", endpoint, t);`

Reply:
I think it will be better with 2 loggers.

Reply:
Yes, I think so, there is possible situation that this exceptionally completed feature can be logged on higher level, but in any case I think we should always log catch causes